spec/models/asset_spec.rb

```diff
@@ -112,7 +112,7 @@
         asset.scan_for_viruses
       }.to change(Delayed::Job, :count).by(1)
 
-      job = Delayed::Job.last
+      job = Delayed::Job.asc('_id').last
```

Comment:
As you explain in the commit note, these `asc('_id')` calls are a bit ugly. I think it would be worth opening a new issue to capture the idea of fixing them otherwise I think they'll just get left.